packages/node/src/indexer/indexer.manager.ts

```diff
@@ -162,15 +163,33 @@ export class IndexerManager {
       ]);
     }
 
-    const lastProcessedHeight = (
-      await this.metadataRepo.findOne({ where: { key: 'lastProcessedHeight' } })
-    ).value as number;
-    void this.fetchService.startLoop(lastProcessedHeight + 1).catch((err) => {
+    let startHeight;
+    const lastProcessedHeight = await this.metadataRepo.findOne({
+      where: { key: 'lastProcessedHeight' },
+    });
+    if (lastProcessedHeight !== null) {
+      startHeight = (lastProcessedHeight.value as number) + 1;
```

Comment:
`       startHeight = Number(lastProcessedHeight.value) + 1;`
in case it is a string.